db/migrate/20160224155909_add_registered_at_to_subscription_facet.rb

```diff
@@ -0,0 +1,9 @@
+class AddRegisteredAtToSubscriptionFacet < ActiveRecord::Migration
+  def up
+    add_column(:katello_subscription_facets, :registered_at, :datetime, :null => false, :default => Time.now)
```

Comment:
Yea, one is in ruby and will default the column to when the migration is run while the other is a sql call and will default the column to when an insert happens.